.nycrc

```diff
@@ -1,13 +1,11 @@
 {
-  "nyc": {
-      "extends": "@salesforce/dev-config/nyc",
-      "check-coverage": true,
-      "lines": 90,
-      "statements": 90,
-      "functions": 90,
-      "branches": 90,
-      "reporter": ["lcov", "text"],
-      "extension": [".js"],
-      "exclude": ["bin/**/*.js", "**/*.test.js"]
-  }
-}
\ No newline at end of file
+  "extends": "@salesforce/dev-config/nyc",
+  "check-coverage": true,
+  "lines": 90,
```

Comment:
does this fix mean it starts applying the 90 now?  how does this interact with https://github.com/forcedotcom/dev-config/pull/3/files ?

Reply:
yes, this repo had a custom `nyc` file - it wasn't in the correct format though. I'm sure we can have it use the default from `dev-config`. It doesn't have any tests anyways 🤷‍♂️